uniswap-v2/logics/traits/pair.rs

```diff
@@ -165,3 +193,85 @@ impl From<ReentrancyGuardError> for PairError {
         PairError::ReentrancyGuardError(error)
     }
 }
+
+#[derive(Default, Debug, Clone, Copy, PartialEq, Eq, Encode, Decode)]
```

Comment:
Can you move this in a type.rs file ? Would be cleaner

Reply:
Moved to types under traits following openbrush's convention, e.g. `use openbrush::traits::AccountId`.